cmd/manager/main.go

```diff
@@ -87,6 +92,17 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Setup event recorder
+	kubeClient, err := kubernetes.NewForConfig(ctrl.GetConfigOrDie())
+	if err != nil {
+		setupLog.Error(err, "error creating kubernetes client")
+		os.Exit(1)
+	}
+	eventBroadcaster := record.NewBroadcaster()
+	logger := zap.New(zap.UseDevMode(true))
+	eventBroadcaster.StartLogging(logger.Info)
+	eventBroadcaster.StartRecordingToSink(&clientv1.EventSinkImpl{Interface: kubeClient.CoreV1().Events("")})
```

Comment:
so, we are going to log all events in a cluster? Do we need this? 
From kubebuilder [documentation](https://book-v1.book.kubebuilder.io/beyond_basics/creating_events.html):
```
Events are published from a Controller using an EventRecorder, which can be created for a Controller by calling GetRecorder(name string) on a Manager.
```
I think it's ok to have
```golang
	if err = (&controllers.AppReconciler{
		TemplateReader: storage,
		Client:         mgr.GetClient(),
		Log:            ctrl.Log.WithName("controllers").WithName("App"),
		Scheme:         mgr.GetScheme(),
		HelmFactoryFn: func(namespace string) (controllers.Helm, error) {
			return chart.NewHelmClient(namespace)
		},
                // no need to create a new Eventbroadcaster 
		Recorder: mgr.GetEventRecorderFor("app"),

```